src/test/ui/trivial-bounds/trivial-bounds-inconsistent-copy.rs

```diff
@@ -3,24 +3,25 @@
 #![feature(trivial_bounds)]
 #![allow(unused)]
```

Comment:
Unnecessary `allow` if the test is changed to `check-pass` (which it should be).